npm/src/cocooned/plugins/core.js

```diff
@@ -31,6 +46,21 @@ const coreMixin = (Base) => class extends Base {
       })
     )
   }
+
+  replacementsFor (association) {
+    return this.constructor.replacementsFor(association);
+  }
+
+  /* Protected and private attributes and methods */
+  static __replacements = [
+    // Default attributes
+    { attribute: 'for', delimiters: ['_'] },
+    { attribute: 'id', delimiters: ['_'] },
+    { attribute: 'name', delimiters: ['[', ']'] },
+
+    // Compatibility with Trix. See #65 on Github.
+    { attribute: 'input', delimiters: ['_'] },
```

Comment:
Is doesn't seem limited to `<trix-editor>` elements. 

I know `input` is a Trix specific html attribute but I'm curious if this could introduce issues with other RTEs or JS plugins which rely on that attribute for another purpose. You'd need a method to opt-out in those cases if this is going to automatically replace `input` attribute on any html element.

Reply:
You're right, even if chances are small it will match with an `input` attribute on another tag *and* find a match with both association name and delimiters in its value and ends doing a wrong replacement.

I changed the `Replacement` class to support an optional `tag` argument in its constructor to restrain matches. It should be fine now.